keyboards/keebmonkey/kbmg68/info.json

```diff
@@ -0,0 +1,80 @@
+                { "label": "K1A (B2,C6)", "x": 9.5, "y": 1 },
+                { "label": "K1B (B2,C7)", "x": 10.5, "y": 1 },
+                { "label": "K1C (B2,F7)", "x": 11.5, "y": 1 },
+                { "label": "K1D (B2,F6)", "x": 12.5, "y": 1 },
+                { "label": "K1E (B2,F5)", "x": 13.5, "y": 1, "w": 1.5 },
+                { "label": "K1F (B2,F4)", "x": 15, "y": 1 },
+                { "label": "K20 (B3,B6)", "x": 0, "y": 2, "w": 1.75 },
+                { "label": "K22 (B3,D0)", "x": 1.75, "y": 2 },
+                { "label": "K23 (B3,D1)", "x": 2.75, "y": 2 },
+                { "label": "K24 (B3,D2)", "x": 3.75, "y": 2 },
+                { "label": "K25 (B3,D3)", "x": 4.75, "y": 2 },
+                { "label": "K26 (B3,D4)", "x": 5.75, "y": 2 },
+                { "label": "K27 (B3,D5)", "x": 6.75, "y": 2 },
+                { "label": "K28 (B3,D6)", "x": 7.75, "y": 2 },
+                { "label": "K29 (B3,D7)", "x": 8.75, "y": 2 },
+                { "label": "K2A (B3,C6)", "x": 9.75, "y": 2 },
+                { "label": "K2B (B3,C7)", "x": 10.75, "y": 2 },
+                { "label": "K2C (B3,F7)", "x": 11.75, "y": 2 },
+                { "label": "K2D (B3,F6)", "x": 12.75, "y": 2, "w": 2.25 },
+                { "label": "K2F (B3,F4)", "x": 15, "y": 2 },
+                { "label": "K31 (B4,B7)", "x": 0, "y": 3, "w": 2.25 },
+                { "label": "K32 (B4,D0)", "x": 2.25, "y": 3 },
+                { "label": "K33 (B4,D1)", "x": 3.25, "y": 3 },
+                { "label": "K34 (B4,D2)", "x": 4.25, "y": 3 },
+                { "label": "K35 (B4,D3)", "x": 5.25, "y": 3 },
+                { "label": "K36 (B4,D4)", "x": 6.25, "y": 3 },
+                { "label": "K37 (B4,D5)", "x": 7.25, "y": 3 },
+                { "label": "K38 (B4,D6)", "x": 8.25, "y": 3 },
+                { "label": "K39 (B4,D7)", "x": 9.25, "y": 3 },
+                { "label": "K3A (B4,C6)", "x": 10.25, "y": 3 },
+                { "label": "K3B (B4,C7)", "x": 11.25, "y": 3 },
+                { "label": "K3D (B4,F6)", "x": 12.25, "y": 3, "w": 1.75 },
+                { "label": "K3E (B4,F5)", "x": 14, "y": 3 },
+                { "label": "K3F (B4,F4)", "x": 15, "y": 3 },
+                { "label": "K40 (B5,B6)", "x": 0, "y": 4, "w": 1.25 },
+                { "label": "K41 (B5,B7)", "x": 1.25, "y": 4, "w": 1.25 },
+                { "label": "K43 (B5,D1)", "x": 2.5, "y": 4, "w": 1.25 },
+                { "label": "K46 (B5,D4)", "x": 3.75, "y": 4, "w": 6.25 },
+                { "label": "K4A (B5,C6)", "x": 10, "y": 4 },
+                { "label": "K4B (B5,C7)", "x": 11, "y": 4 },
+                { "label": "K4C (B5,F7)", "x": 12, "y": 4 },
+                { "label": "K4D (B5,F6)", "x": 13, "y": 4 },
+                { "label": "K4E (B5,F5)", "x": 14, "y": 4 },
+                { "label": "K4F (B5,F4)", "x": 15, "y": 4 }
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```

Comment:
```suggestion
```

Reply:
These comments indicate how this file was generated.  I'm giving credit where credit is due.